RELEASE-NOTES.md

```diff
@@ -1,5 +1,35 @@
 # Prefect Release Notes
 
+## Release 2.4.3
+
+### Enhancements
+- Warn if user tries to login with API key from Cloud 1 — https://github.com/PrefectHQ/prefect/pull/6958
+- Improve concurrent task runner performance — https://github.com/PrefectHQ/prefect/pull/6948
+- Raise a `MissingContextError` when `get_run_logger` is called outside a run context — https://github.com/PrefectHQ/prefect/pull/6980
+- Cache configuration lookups — https://github.com/PrefectHQ/prefect/pull/6959
+- Move `quote` to `prefect.utilities.annotations` — https://github.com/PrefectHQ/prefect/pull/6993
+- Add state filters and sort-by to the work-queue, flow and deployment pages — https://github.com/PrefectHQ/prefect/pull/6985
+
+### Fixes
+- Fix login to private Docker registries — https://github.com/PrefectHQ/prefect/pull/6889
+- Update `Flow.with_options` to actually pass retry settings to new object — https://github.com/PrefectHQ/prefect/pull/6963
+- Fix compatibility for protected blocks when client/server versions are mismatched — https://github.com/PrefectHQ/prefect/pull/6986
+- Ensure `python-slugify` is always used even if [unicode-slugify](https://github.com/mozilla/unicode-slugify) is installed — https://github.com/PrefectHQ/prefect/pull/6955
+- Fix configuration caching — https://github.com/PrefectHQ/prefect/pull/7003
+- Add block registration to models init — https://github.com/PrefectHQ/prefect/pull/7008
+
+### Documentation
+- Update docs regarding schedules and the CLI — https://github.com/PrefectHQ/prefect/pull/6968
+- Fix deployment file name in the tutorial — https://github.com/PrefectHQ/prefect/pull/6956
+- Add results concept to documentation — https://github.com/PrefectHQ/prefect/pull/6992
+
+### Collections
+- Adds prefect hex to catalog — https://github.com/PrefectHQ/prefect/pull/6974
```

Comment:
```suggestion
- Add prefect-hex to Prefect Collections catalog — https://github.com/PrefectHQ/prefect/pull/6974
```

Reply:
@tpdorsey can you submit this suggestion off the latest commit?

Reply:
Ah yeah I changed this a bit already